packages/stepper/src/EllipsesStep/index.tsx

```diff
@@ -0,0 +1,47 @@
+import Tooltip, { Align, Justify } from '@leafygreen-ui/tooltip';
```

Comment:
Undecided on whether I prefer using `index.tsx` here, vs `EllipsesStep.tsx`, or both files

I see the benefits of this version (import links are shorter), but it makes it harder to find files and see which file you're working on. In VSCode you can find a specific file (`CMD-T` I think). Having these all named index makes this harder. It's also harder to see which file you have open. E.x. If I have `EllipsesStep/index` and `InternalStep/index` open it takes a bit more attention to see which file I'm in

Reply:
Yeah this is the tail end of the discussion your pupper took you away from 😂

Dave and I agreed with you, and I'll be raising POC PRs with that change applied.